sgt-back-end/server.js

```diff
@@ -0,0 +1,171 @@
+import express from 'express';
+import pg from 'pg';
+
+const app = express();
+
+app.use(express.json());
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', async (req, res) => {
+  try {
+    const sql = `
+      select *
+        from "grades"
+    `;
+    const result = await db.query(sql);
+    res.status(200).json(result);
+  } catch (err) {
+    console.error(err);
+    res.status(500).json({ error: 'An unexpected error occurred.' });
+  }
+});
+
+app.post('/api/grades', async (req, res) => {
+  try {
+
+    let error = {};
+    const keyWords = ['name', 'course', 'score'];
+    const input = req.body;
+    const name = req.body.name;
+    const course = req.body.course;
+    const score = Number(req.body.score);
+
+    for (const keys of keyWords) {
+      if (!input[keys]) {
+        error = { error: `please include key word ${keys}` };
+        res.status(400).send(error);
+        return;
+      } else if (score < 0) {
+        error = { error: 'please ensure score is a positive number' };
+        res.status(400).send(error);
+        return;
+      } else if (score > 100) {
+        error = { error: 'please ensure score is not over 100' };
+        res.status(400).send(error);
+        return;
+      }
+    }
+    if (isNaN(score)) {
```

Comment:
You should probably do this check that verifies `score` is a number, before you compare it against 0 and 100. Otherwise you will be comparing a string, which is weird!